cmd/template.go

```diff
@@ -47,6 +47,7 @@ func NewTemplateCmd(globalCfg *config.GlobalImpl) *cobra.Command {
 	f.StringVar(&templateOptions.PostRenderer, "post-renderer", "", `pass --post-renderer to "helm template" or "helm upgrade --install"`)
 	f.StringArrayVar(&templateOptions.PostRendererArgs, "post-renderer-args", nil, `pass --post-renderer-args to "helm template" or "helm upgrade --install"`)
 	f.StringVar(&templateOptions.KubeVersion, "kube-version", "", `pass --kube-version to "helm template". Overrides kubeVersion in helmfile.yaml`)
+	f.StringArrayVar(&templateOptions.ShowOnly, "show-only", nil, `pass --show-only to "helm template"`)
```

Comment:
@jim-barber-he Thank you very much for the detailed feature request and this PR. I believe this will be a useful addition to Helmfile!

This isn't strictly related to show-only only but just wanted to confirm; How do you use this flag when you have many releases in helmfile.yaml and you want to set show-only only on a subset of releases?

Reply:
If I understand your question properly, then where you have multiple releases with multiple charts, you need to set the selector accordingly as well.
e.g.
```bash
helmfile --selector name=$RELEASE_NAME template --skip-deps --show-only templates/config-files.yaml
```

I've just tried running it without a selector like so (using `--args` because the helmfile binary I'm using hasn't got my change):
```
helmfile template --skip-deps --args '--show-only templates/config-files.yaml'
```
And it ends up throwing an error when looking at a release whos chart doesn't have `templates/config-files.yaml` in it.
```
  Error: could not find template templates/config-files.yaml in chart
```